generator/internal/rust_prost/templates/prost/Cargo.toml.mustache

```diff
@@ -0,0 +1,37 @@
+{{!
+Copyright 2025 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+}}
+# Copyright {{CopyrightYear}} Google LLC
+{{#BoilerPlate}}
+#{{{.}}}
+{{/BoilerPlate}}
+
+[package]
+name        = "{{PackageName}}-code-generation"
+version     = "0.0.0"
+description = "Prost and Tonic generator for {{PackageName}}"
+publish     = false
+build       = "build.rs"
+edition     = "2021"
+
+[build-dependencies]
+prost-build = { version = "0.13", optional = true }
+tonic-build = { version = "0.12", optional = true }
+
+[features]
+_generate-protos = ["dep:prost-build", "dep:tonic-build"]
```

Comment:
just curious, why the leading underscore?

Reply:
https://github.com/rust-lang/cargo/issues/10882#issuecomment-1880844632

Features with leading underscores are hidden from the documentation. Seems like a neat idea to adopt for features we do not want to advertise. Either way would be mostly harmless in these crates, as they will never be published.